sspl_test/common.py

```diff
@@ -58,7 +58,7 @@
 MODULES         = 'modules'
 SYS_INFORMATION = 'SYSTEM_INFORMATION'
 PRODUCT_NAME    = 'product'
-
+conf_reader = None
```

Comment:
What is the use of this initialization?

Reply:
Because conf_reader is still passed in initialize method of every class